tests/interop/go-tuf/generate.go

```diff
@@ -0,0 +1,19 @@
+package main
```

Comment:
I don't think this will build because it'll pull the wrong go-tuf. Instead of leaving this here, maybe point to the commit from upstream that these were taken from? It makes more sense to generate locally and then copy out anyway IMO.

Also, that implies that you should probably call this directory M3, since nothing will keep it in sync with go-tuf's master; all of these directories are snapshots.

Reply:
Good idea.

Regarding the name, I think we should also link it to our fork, so I prefixed the directory with `fuchsia-`. I don't want to use M3 though, since this metadata isn't what we called M3. I instead named this directory `fuchsia-go-tuf-HEAD`, and called out the specific commit in the README. That sound good?